colab/tflite_text_classification.ipynb

```diff
@@ -0,0 +1,514 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Copy of tflite_text_classification.ipynb",
+      "provenance": [],
+      "collapsed_sections": [
+        "dweksnWs1eZb"
+      ]
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_VsBo9D7GJ8G"
+      },
+      "source": [
+        "```\n",
+        "Copyright 2020 The IREE Authors\n",
```

Comment:
Copyright year 2021

```suggestion
        "Copyright 2021 The IREE Authors\n",
```

I'm also tempted to switch these back to the collapsed license header format that we had before (copied from TF). WDYT?

before: 
![image](https://user-images.githubusercontent.com/4010439/120378012-bc1efc00-c2d2-11eb-8897-31e0d7210354.png)

after:
![image](https://user-images.githubusercontent.com/4010439/120378034-c3460a00-c2d2-11eb-852f-94136ffd4ba5.png)

here's what TF does ([notebook](https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/quickstart/beginner.ipynb)):
![image](https://user-images.githubusercontent.com/4010439/120378118-e07ad880-c2d2-11eb-98dd-72b99de65dc4.png)

which expands to this:
![image](https://user-images.githubusercontent.com/4010439/120378272-0f914a00-c2d3-11eb-92fc-15e59ce7541a.png)

Reply:
I think it seems okay for now to leave the new header so it is consistent with the other colab notebooks, but in a followup PR I think it would be good to update all of them so that they are collapsed